src/components/Home/index.tsx

```diff
@@ -88,6 +88,18 @@ const Overview: React.FC<{
           linkLabel="View Logs"
           testId="emulator-info-functions"
         />
+        <EmulatorCard
```

Comment:
I think we should link to the hosted website at host:port with a "View website" label.

Reply:
👍